<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hra</class>
 <widget class="QMainWindow" name="Hra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="spravneText">
        <property name="text">
         <string>Správně: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="celkemText">
        <property name="text">
         <string>Celkem: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spatneText">
        <property name="text">
         <string>Špatně: 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="platnoGV"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="spravneBtn">
        <property name="text">
         <string>Správně</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="spatneBtn">
        <property name="text">
         <string>Špatně</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
